kevinbabanats/GBC_Train_Program
# added documentation and ev3 support

This PR introduces support for the LEGO EV3 Brick in our project and adds documentation to guide users through the setup process. The main changes include:

Documentation:

Added a detailed Markdown document outlining the setup process for using Bluetooth communication with the LEGO EV3 Brick.
Included instructions for hardware setup, software installation, and code implementation.
EV3 Support:

Added Python code to handle Bluetooth communication, motor control, and sensor reading for the LEGO EV3 Brick.
Integrated the Bluetooth serial communication method to receive data and control motors accordingly.

The code provided in this PR has not been tested. Users are advised to test the implementation in their own environment and make necessary adjustments as needed.
How to Test:

Follow the setup instructions in the ev3_setup.md documentation.
Ensure that your LEGO EV3 Brick is properly configured and paired with your Bluetooth device.
Run the ev3_bluetooth.py script and verify Bluetooth communication and motor control functionality.
Additional Information:

Ensure that the Bluetooth MAC address used in the script matches your EV3 Brick's MAC address.
Verify that all required Python libraries are installed and up to date.
